sound/soc/codecs/hdac_hda.c

```diff
@@ -15,7 +15,6 @@
 #include <sound/soc.h>
 #include <sound/hdaudio_ext.h>
 #include <sound/hda_i915.h>
-#include <sound/hda_codec.h>
```

Comment:
this file also uses struct definitions and function declarations from the header, so it should keep it too, not rely on it being pulled in by another header.

Reply:
aa, you are right @lyakh, will change